AdminWebsite/AdminWebsite/ClientApp/src/app/booking/create-hearing/create-hearing.component.html

```diff
@@ -64,23 +64,44 @@ <h2 class="govuk-error-summary__title" id="error-summary-title">Please complete
 
   <div *ngIf="availableCaseTypes.length > 1" [ngClass]="caseTypeInvalid ? 'govuk-form-group--error' : 'govuk-form-group'">
     <label class="govuk-label govuk-!-width-one-half" for="caseType"> Case type </label>
-    <select class="govuk-select govuk-!-width-one-half" id="caseType" formControlName="caseType">
+    <!--suppress XmlDuplicatedId -->
+    <select *ngIf="!this.editMode" class="govuk-select govuk-!-width-one-half" id="caseType" formControlName="caseType">
```

Comment:
shouldn't this be the other way round? If in edit mode, show the dropdown, else show a text box?

Reply:
I think the `EditMode` property might be misleading. This means edit an already booked hearing. You can only choose these values once upon booking, not after, so the drop down is available whilst you're making the booking, not when you're editing

Reply:
ah right ok that makes sense. Cheers